src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,13 +1,13 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.dataStructures.IntArraySTack;
+import edu.kis.vh.nursery.dataStructures.IntArrayStack;
 
 public class DefaultCountingOutRhymer {
-    public IntArraySTack arraySTack = new IntArraySTack();
+    public IntArrayStack arraySTack = new IntArrayStack();
 
     public DefaultCountingOutRhymer(){}
 
-    public DefaultCountingOutRhymer(IntArraySTack arraySTack) {
+    public DefaultCountingOutRhymer(IntArrayStack arraySTack) {
         this.arraySTack = arraySTack;
```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable. You should use the common interface where possible!!!